assets/control-plane/prometheus-rule.yaml

```diff
@@ -392,21 +396,21 @@ spec:
         severity: warning
   - name: kubernetes-system-apiserver
     rules:
-    - alert: AggregatedAPIErrors
+    - alert: KubeAggregatedAPIErrors
```

Comment:
we should make sure that we advertise this change in the release notes :)

Reply:
Can I do it in a followup PR?

/hold

Reply:
@simonpasquier I see that a run book exists for [AggregatedAPIErrors](https://github.com/openshift/runbooks/blob/master/alerts/AggregatedAPIErrors.md), but looks the isn't referenced from [cmo run book patching ](https://github.com/openshift/cluster-monitoring-operator/blob/9534afc81bc1f687bcabf3183f878a338d604f55/jsonnet/utils/sanitize-rules.libsonnet#L335).

Reply:
> Can I do it in a followup PR?
@simonpasquier changelog has been updated to reflect this rename.

Reply:
> @simonpasquier I see that a run book exists for AggregatedAPIErrors, but looks the isn't referenced from cmo run book patching .

IIUC we need to create a new entry in openshift/runbooks + add `KubeAggregatedAPIErrors` to the runbooks patching. But I agree it can be done in a second PR.